src/dispatch/cli.py

```diff
@@ -161,21 +161,24 @@ def sync_triggers():
     from sqlalchemy_searchable import sync_trigger
 
     sync_trigger(engine, "definition", "search_vector", ["text"])
+    sync_trigger(engine, "dispatch_user", "search_vector", ["email"])
     sync_trigger(engine, "document", "search_vector", ["name"])
     sync_trigger(engine, "incident", "search_vector", ["name", "title", "description"])
+    sync_trigger(engine, "incident_cost_type", "search_vector", ["name", "description"])
+    sync_trigger(engine, "incident_priority", "search_vector", ["name", "description"])
     sync_trigger(engine, "incident_type", "search_vector", ["name", "description"])
     sync_trigger(
         engine, "individual_contact", "search_vector", ["name", "title", "company", "notes"]
     )
+    sync_trigger(engine, "notification", "search_vector", ["name", "description"])
     sync_trigger(engine, "plugin", "search_vector", ["title"])
-    sync_trigger(engine, "search_filter", "search_vector", ["name", "description"])
     sync_trigger(engine, "report", "search_vector", ["details_raw"])
+    sync_trigger(engine, "search_filter", "search_vector", ["name", "description"])
     sync_trigger(engine, "service", "search_vector", ["name"])
     sync_trigger(engine, "tag", "search_vector", ["name"])
     sync_trigger(engine, "task", "search_vector", ["description"])
     sync_trigger(engine, "team_contact", "search_vector", ["name", "company", "notes"])
     sync_trigger(engine, "term", "search_vector", ["text"])
-    sync_trigger(engine, "dispatch_user", "search_vector", ["email"])
     sync_trigger(engine, "workflow", "search_vector", ["name", "description"])
```

Comment:
@kevgliss I added sync_trigger() calls for some that were missing. I didn't check all models, so there could be more.